src/controller/proxy/local.go

```diff
@@ -217,4 +218,5 @@ func SendPullEvent(a *artifact.Artifact, tag, operator string) {
 		Operator: operator,
 	}
 	event.BuildAndPublish(e)
+	util.UpdatePullTime(context.Background(), e.Artifact.ID, e.Tag)
```

Comment:
The event handler will update the pull time automatically, why do you need to call the function to do the same work here?

Reply:
Updated